crypto/mod.ts

```diff
@@ -133,6 +134,7 @@ const stdCrypto = ((x) => x)({
         return fnv(algorithm.name, bytes);
       }
 
+      const wasmCrypto = instantiateWasm();
```

Comment:
Important to note that this is now instantiated lazily and not up front, so the first call to any code path that instantiates a module will be slower than before.